packages/crud/src/vaadin-crud-grid.js

```diff
@@ -109,10 +109,28 @@ class CrudGrid extends IncludedMixin(Grid) {
    */
   _configure(item) {
     this.innerHTML = '';
+    this.__itemPropertyDepth = this.__getPropertyDepth(item);
     this.__createColumns(this, item);
     this.__toggleEditColumn();
   }
 
+  /**
+   * Return the deepest property depth of the object
+   * @private
+   **/
```

Comment:
nit: let's fix a comment (we need a ESLint rule for it).

```suggestion
   */
```